### PR TITLE
First draft of UI elements lesson

### PR DESCRIPTION
**Pull request by mdippery**
cc: @JimCampagno @AnnJohn

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,20 +19,121 @@
...
+## UIKit
+
+iOS includes a framework called _UIKit_. UIKit provides you with a standard set of widgets, referred to as _elements_ or _controls_, that you can use in your app's views to provide a way for your users to interact with your app.
+
+## UI Controls
+
+So far, you have been introduced to a few user interface elements, including basic views, labels, and buttons. In this lesson, you'll get a quick overview of some of the other interesting user interface elements that you can use in your iOS apps.
+
+To begin this investigation, create a new iOS project in Xcode. You can use the single-view application template provided with Xcode. (There is also a sample application included in this repository. It's located at `UIElements/UIElements.xcodeproj`. But you should start your own project so you can get a feel for these elements yourself.)
+
```
Comment (JimCampagno) [inline README.md:34]: 👍

File: README.md [hunk 2]
```diff
@@ -21,20 +19,121 @@
...
+### Labels
+
+_Labels_, also known as `UILabel`s, are the most basic way to display strings of text in your app. You've already seen a few labels in your previous lessons, and should be pretty comfortable with them by now. Just for old time's sake, go ahead and drag a label onto your app's main view. You can find it in the object library in the bottom left corner of Xcode's main window. (You _do_ remember where that is and how to filter for labels, right?) Drop it in the top left corner of the application, so it looks like this:
+
```
Comment (JimCampagno) [inline README.md:38]: I think you mean "You can find it in the object library in the bottom right corner of Xcode's main window."

File: README.md [hunk 3]
```diff
@@ -21,20 +19,121 @@
...
+### Sliders
+
+iOS apps often have sliders, too. Here's what they look like:
+
+![Sliders](https://upload.wikimedia.org/wikipedia/commons/d/d2/A_party_tray_of_sliders_at_a_restaurant.jpg)
+
+Just kidding. We're not talking about the little hamburgers you get from White Castle. A _slider_ is a control that has a range of values that the user can _slide_ between. For example, you can use a slider to allow the user to set a duration, or perhaps change colors within a certain range. They look pretty cool, too. Drag a slider from the object library and position it under your text field, so your UI looks like this:
+
+![Horizontal sliders](http://i.imgur.com/4Tt1ckZ.png)
+
```
Comment (JimCampagno) [inline README.md:86]: lol! Wonderful

**Comment by JimCampagno** [comment]
@mdippery This is exactly how I wanted this lesson to look like. I like giving the examples of HOW and WHY they would be using these various elements, that should go a long way. I can also see a student visiting this reading from time to time to get the basics under their belt again.

File: README.md (hunk 2 above)
Comment (mdippery) [inline README.md:38]: I did! Fixed.

**Comment by mdippery** [comment]
Thanks, Jim. Lot of fun writing this one, too!

**Comment by JimCampagno** [comment]
@mdippery Perfecto!

**Pull request merged**